src/syncedEnforcer.ts

```diff
@@ -0,0 +1,520 @@
+// Copyright 2020 The Casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Enforcer, newEnforcerWithClass } from './enforcer';
```

Comment:
I remand we added a syncedEnforcer, if we removed old syncedEnforcer, we need to restore this.

Reply:
You mean we should keep the git history?

Reply:
This is exactly same as the old syncedEnforcer, it was previous removed at https://github.com/casbin/casbin.js/commit/7ca1f2616f8f474ba521e8e1f8ac17b47c98b22a
I just add support for sync role manager

Reply:
syncedEnforcer  reverted by #159.